policies/terms_of_membership.md

```diff
@@ -0,0 +1,113 @@
+
+### Noise
+
+It is our responsibility to be good neighbours. The people living in the area are used to a quiet neighbourhood and we will do everything we can to ensure it stays that way.
+
+- Please ensure that noise made cannot be heard outside the building. 
+- Please ensure that you are quiet when entering and leaving the building. The gates can be very noisy!
+
+Normal working hours are 9am to 5pm. Whilst it is reasonable to expect some level of noise during these hours, every effort must be made to reduce noise outside of that time.
+
+### Child protection and Safeguarding
+
+The document in the link below outlines the safeguarding policy and procedures for all work undertaken with children under the age of 18, and vulnerable adults at Farset Labs, in accordance with Children’s (NI) Order 1995, and Our Duty to Care, Child Care NI 1995.
+
+- As a member you agree to read and implement this policy in full.
+
+[Child Protection](child_protection.md)
+
+## Respect
+
+We aim to be good members of the local community, who are important stakeholders of the charity. 
+
+Within our own community, we all need to strive towards cohesion and harmony. Therefore we expect our members to have respect for each other. We must listen and communicate with consideration for our fellow members. Please try to settle disputes amicably and be prepared to listen to the concerns other members might have. Think carefully about your fellow members' privacy, values, opinions and wellbeing.
+
+The trustees will try and mediate disputes should that become necessary. 
+
+## Culture and Ethos
+
+ * Be Excellent to each other;
+ * Don't disturb those who do not want to be disturbed;
+ * If you see something and it doesn't look right or safe, it probably isn't so ask/tell someone about it;
+ * [Intolerance will not be tolerated](equality.md).
+
+## Handling of Complaints and Breaches
+
+ * All members should endeavour to resolve minor personal issues directly with the persons involved;
+ * If this is not possible, the following general approach is to be applied;
+   * Raise a complaint to admin@farsetlabs.org.uk detailing the issue and the persons involved. Reports may be anonymously filed but this will slow the response, as the board will first have to corroborate the claims raised.
+   * The issue trigger an Extraordinary Board meeting meeting and, at that boards discretion, directly with the persons involved.
+   * After at most two weeks, a response will be issued by a member of the management board that may result in a warning or subsequent expulsion of those involved.
+   * In the case of "minor infractions", a right to reply to the judgements of the board is permitted, and the board will respond to reasonable requests for explanation.
+   * The board reserves the right to unilaterally and without any right to reply to permanently ban anyone who, through negligence or malice, endangers or risks endangering guests, members, the facility, or the charity. 
```

Comment:
Widen the scope of this to clarify that the board can ban anyone who endangers others, **or anyone who breaches these rules.** Potentially also include a clause on harassment etc